packages/design-system/src/components/MonthPicker/MonthPicker.tsx

```diff
@@ -139,6 +139,10 @@ export const MonthPicker = (props: MonthPickerProps) => {
   const { hintId, hintElement } = useHint({ ...props, id });
   const labelProps = useLabelProps({ ...props, id });
 
+  // 🥑
+  // https://www.w3.org/TR/wai--1.2/#group
+  //    - "aria-invalid (state) (deprecated on this role in ARIA 1.2)"
+
   return (
     <fieldset
```

Comment:
TODO: Add `role="listbox"` as possible remediation of deprecated `aria-invalid` attr on group roles.